test-d/jsonify.ts

```diff
@@ -192,6 +192,9 @@ expectType<never>(plainSymbol);
 declare const arrayMemberUndefined: Jsonify<Array<typeof undefined>>;
 expectType<null[]>(arrayMemberUndefined);
 
+declare const arrayMemberUnionWithUndefined: Jsonify<Array<typeof undefined | typeof number>>;
+expectType<Array<null | number>>(arrayMemberUnionWithUndefined);
```

Comment:
Can you also do a test where `Array<typeof undefined | typeof number>` is not at the top-level?

Reply:
Added!